src/EnergyPlus/StateManagement.cc

```diff
@@ -404,6 +404,7 @@ void EnergyPlus::clearAllStates()
     UnitHeater::clear_state();
     UnitVentilator::clear_state();
     UserDefinedComponents::clear_state();
+    UtilityRoutines::clear_state();
```

Comment:
Very nice.  There was a time when `clear_states` were being added almost daily, but now it's a very rare thing for a namespace/module to _not_ have one.  This is good.